iothub/service/src/DigitalTwin/DigitalTwinClient.cs

```diff
@@ -19,6 +19,7 @@ namespace Microsoft.Azure.Devices
 {
     /// <summary>
     /// The Digital Twins Service Client contains methods to retrieve and update digital twin information, and invoke commands on a digital twin device.
+    /// For more information, see <see href="https://github.com/Azure/azure-iot-sdk-csharp/tree/previews/RBAC#iot-hub-service-sdk"/>
```

Comment:
be careful, that link is to our preview branch. I'd use a link to the master branch instead